example/airship-example.cabal

```diff
@@ -25,3 +26,25 @@ executable example
                       , unordered-containers
                       , wai == 3.0.*
                       , warp == 3.0.*
+
+executable            versions
+  main-is:              Versions.hs
```

Comment:
It's really just an executable version of this https://github.com/helium/airship/blob/master/doc/versioning-apis.md#by-content-type
that gets compiled and checked it still works.

Splitting small pieces out, as here's how to do a POST or how to handle versions, is my preferred way of looking at example code. Seems especially important for this since you can assemble non-sense resources that don't do what you expect, but still compile.